<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reg</class>
 <widget class="QWidget" name="reg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40,49,59);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="btn_login_2">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none; 
color: rgb(220, 220, 220);</string>
       </property>
       <property name="text">
        <string>Войти</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_registr">
       <property name="styleSheet">
        <string notr="true">
border: none; 
border-bottom: 3px solid rgb(97, 190, 46); 
color: rgb(220, 220, 220);
</string>
       </property>
       <property name="text">
        <string>Регистрация</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>7</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="label1">
         <property name="styleSheet">
          <string notr="true">color: rgb(220, 220, 220);</string>
         </property>
         <property name="text">
          <string>Логин:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_login_2">
         <property name="styleSheet">
          <string notr="true">color: rgb(220, 220, 220);</string>
         </property>
         <property name="maxLength">
          <number>1000</number>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label2">
         <property name="styleSheet">
          <string notr="true">color: rgb(220, 220, 220);</string>
         </property>
         <property name="text">
          <string>Эл. почта:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_email">
         <property name="styleSheet">
          <string notr="true">color: rgb(220, 220, 220);</string>
         </property>
         <property name="maxLength">
          <number>32768</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label3">
         <property name="styleSheet">
          <string notr="true">color: rgb(220, 220, 220);</string>
         </property>
         <property name="text">
          <string>Пароль:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_password_2">
         <property name="styleSheet">
          <string notr="true">color: rgb(220, 220, 220);</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label4">
         <property name="styleSheet">
          <string notr="true">color: rgb(220, 220, 220);</string>
         </property>
         <property name="text">
          <string>Подтверждение:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_confirm_pass">
         <property name="styleSheet">
          <string notr="true">color: rgb(220, 220, 220);</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="btn_reg">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(117,174,93);
color: rgb(40,49,59);</string>
         </property>
         <property name="text">
          <string>Регистрация</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
